services/src/main/java/org/keycloak/authentication/AuthenticationProcessor.java

```diff
@@ -82,6 +82,7 @@ public class AuthenticationProcessor {
     public static final String BROKER_SESSION_ID = "broker.session.id";
     public static final String BROKER_USER_ID = "broker.user.id";
     public static final String FORWARDED_PASSIVE_LOGIN = "forwarded.passive.login";
+    public static final String USER_SESSION_ID = "user.session.id";
```

Comment:
Not sure this is the best place to put this constant, I thought about adding a Constants class in services module, but dropped the idea at the end.